docs/notes.rst

```diff
@@ -1,8 +1,42 @@
-Version 1.8.1
+Version 1.8.2
 =============
 
 This release includes the following features:
 
+* Modified unknown writers default behavior.
+* Multiple initial PDP announcements.
+* Flag to avoid builtin multicast.
+* *STRICT_REALTIME* compilation flag.
+
+It also adds the following bug fixes and improvements:
+
+* Fix for setting `nullptr` in a fixed string.
+* Fix for not sending GAP in several cases.
+* Solve *Coverity* report issues.
+* Fix issue of *fastrtpsgen* failing to open *IDL.g4* file.
+* Fix unnamed lock in *AESGCMGMAC_KeyFactory.cpp*.
+* Improve *XMLProfiles* example.
+* Multicast is now sent through *localhost* too.
+* *BitmapRange* now implements sliding window.
+* Improve *SequenceNumber_t* struct.
+* Participant's liveliness is now asserted when receive any RTPS message.
+* Fix leak on RemoteParticipantLeaseDuration.
+* Modified default values to improve behavior in *Wi-Fi* scenarios.
+* *SubscriberHistory* improvements.
+* Removed use of *acceptMsgDirectTo*.
+* *WLP* improvements.
+
+**Note:** If you are upgrading from a version older than 1.7.0, it is **required** to regenerate generated source
+from IDL files using *fastrtpsgen*
```

Comment:
I miss:

- Performance test improvements -> I assume this is intentional, as we said we wouldn't mention changes to tests
- Fix valgrind uninitialized error #743 -> not sure this is big enough to be mentioned
- Added copy constructor to ResourceLimitedVector -> not sure this is big enough to be mentioned

Reply:
> I miss:
> 
> * Performance test improvements -> I assume this is intentional, as we said we wouldn't mention changes to tests

Yes, it was intentional.

> * Fix valgrind uninitialized error #743 -> not sure this is big enough to be mentioned

This error was introduced between 1.8.1 and 1.8.2, so isn't a fix from the previous version, ti was a "development phase" error.

> * Added copy constructor to ResourceLimitedVector -> not sure this is big enough to be mentioned

`ResourceLimitedVector` is an internal class, not visible to the user. Am I wrong?

Reply:
You're right. But do we only mention changes visible to the user? Maybe in this case it is not a big deal, so let's just not mention it.